playbook/app/pb_kits/playbook/pb_highlight/_highlight.tsx

```diff
@@ -3,33 +3,40 @@ import Highlighter from 'react-highlight-words'
 import React from 'react'
 import classnames from 'classnames'
 import { globalProps, GlobalProps } from '../utilities/globalProps'
+import { Colors } from '../types'
 
 type HighlightProps = {
+  backgroundColor?: Colors | 'yellow' | 'blue',
   className?: string,
   data?: {[key: string]: string},
   id?: string,
   children?: React.ReactChild[] | React.ReactChild | string,
   text?: string,
+  textColor?: Colors,
   highlightedText?: string[],
 } & GlobalProps
 
 const Highlight = (props: HighlightProps): React.ReactElement => {
   const {
+    backgroundColor = 'yellow',
     children,
     className = 'pb_highlight_kit',
     data = {},
     highlightedText = ['highlight'],
     id = '',
     text = '',
+    textColor = 'black',
   } = props
 
   const highlightContent: any = text || children;
+  const backgroundColorClass = `highlight_bg_color_${backgroundColor}`
+  const textColorClass = `highlight_text_color_${textColor}`
 
   return (
     <Highlighter
         autoEscape
         data={data}
-        highlightClassName={classnames(globalProps(props), className)}
+        highlightClassName={classnames(className, backgroundColorClass, textColorClass, globalProps(props))}
```

Comment:
Code looks great! Small thing, but I think className needs to be the last item passed to classnames for it to work. cc: @jasperfurniss